docs/docs/reference/release-notes/image-migration-guide.md

```diff
@@ -0,0 +1,232 @@
+- How-To Guide: [Using the beta Gatsby Image plugin](/docs/how-to/images-and-media/using-gatsby-plugin-image/)
+- Reference Guide: [Gatsby Image plugin](/docs/reference/built-in-components/gatsby-plugin-image/)
+
+## What changed?
+
+### New Syntax
+
+The new plugin requires significant syntax changes. We've provided a [codemod](#codemod-instructions) to help you migrate, but this section will explain the changes.
+
+#### Import change
+
+Previously, `GatsbyImage` was the default export from `gatsby-image`. With `gatsby-plugin-image`, the component is now a named export.
+
+```javascript
+// import Img from "gatsby-image"
+import { GatsbyImage } from "gatsby-plugin-image"
+```
+
+#### GraphQL changes
+
+This is one of the bigger changes. There are no more fragments to use. Instead, things like `layout` and `format` are passed as arguments to the resolver.
+
+This is the previous syntax.
+
+```javascript
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  {
+    file(relativePath: { eq: "images/example.jpg" }) {
+      childImageSharp {
+        fixed {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
+```
+
+The new syntax looks like this. The fragment is removed in favor of `gatsbyImageData`, which is used for all images. Previous differences - such as `fixed` vs. `fluid`, WebP generation, and placeholder type - are passed as arguments to the resolver.
+
+```javascript
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  {
+    file(relativePath: { eq: "images/example.jpg" }) {
+      childImageSharp {
+        gatsbyImageData(layout: FIXED)
+      }
+    }
+  }
+`
+```
+
+Other changes to the available argument structure are in the section on [API changes](#api-changes).
+
+#### Component changes
+
+The last change is to the JSX component. The import name is potentially different, and the query result is also different.
+
+```javascript
+// import Img from "gatsby-image"
+import { GatsbyImage } from "gatsby-plugin-image"
+
+const HomePage = ({ data }) => {
+  return (
+    // <Img fixed={data.file.childImageSharp.fixed} />
+    <GatsbyImage image={data.file.childImageSharp.gatsbyImageData} />
+  )
+}
+```
+
+### API Changes
+
+In addition to the syntax changes for using `gatsby-plugin-image`, there are also changes to the API that affect the resolver arguments (and `StaticImage`).
+
+#### fluid
+
+The `fluid` image type has been deprecated in favor of two alternatives.
+
+The first is an image type called `fullWidth`. This image is designed to be used for things like hero images and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. In most cases you will not need to provide these, as there are default values that work for most sites. Note that `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
```

Comment:
```suggestion
The first is an image type called `fullWidth`. This image is designed to be used for things like hero images that span the full width of the screen, and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. In most cases you will not need to provide these, as there are default values that work for most sites. Note that like the old `fluid` layout, `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
```